documentation/book/con-metrics-prometheus-resources.adoc

```diff
@@ -15,3 +15,4 @@ When you apply the Prometheus configuration, the following resources are created
 * A `Prometheus` to manage the configuration of the Prometheus pod.
 * A `PrometheusRule` to manage alerting rules for the Prometheus pod.
 * A `Secret` to manage additional Prometheus settings.
+* A `Service` to provide an easy to reference hostname for other services to connect to Prometheus.
```

Comment:
"to allow applications running in the cluster to connect to Prometheus (i.e. Grafana using Prometheus as datasource)"